Config.hs

```diff
@@ -20,8 +20,7 @@ nixSources = do
           package ("feeluown-" <> x)
             `fromPypi` ("fuo_" <> unPkgName x)
 
-  -- https://github.com/NixOS/nixpkgs/issues/119284
-  def $ package "feeluown-core" `sourceManual` "3.7.5" `fetchPypi` "feeluown"
+  def $ package "feeluown-core" `sourceManual` "3.7.6" `fetchPypi` "feeluown"
```

Comment:
Please uncomment the next line to make it track the latest version, instead of specifying 3.7.6 manually

Reply:
@berberman PTAL